src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
> The problem with *not* adding sugar to common functions is that I now have to write `R.eqBy(R.propEq('something'))` in every place where I used to have `R.propEq`

Let's be fair with our comparison:

```javascript
// before
R.eqProps('something')

// after
R.eqBy(R.prop('something'))
```

The latter doesn't seem too bad to me. In fact, I find it clearer.